apps/publish/__init__.py

```diff
@@ -68,15 +71,33 @@ def init_app(app):
     service = UnpublishService(endpoint_name, backend=get_backend())
```

Comment:
this should be Any as well

Reply:
what do you mean?

Reply:
it must be set only when service is first declared in the function